protocol.html

```diff
@@ -559,6 +572,40 @@ <h3 property="schema:name">Resource Containment</h3>
+
+                  <section id="authoritative-resource-data" inlist="" rel="schema:hasPart" resource="#authoritative-resource-data">
+                    <h4 property="schema:name">Authoritative Resource Data</h4>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>Container descriptions are not limited to containment triples. To further support client navigation and application interaction, servers can include <a href="#authoritative-information">authoritative information</a> about contained resources as part of the container description.</p>
+
+                      <p><span about="" id="authoritative-contained-resource-data" rel="spec:requirement" resource="authoritative-contained-resource-data"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> include the following information about contained resources as part of the container description, unless that information does not exist or is expensive to determine:</span></span></p>
+
+                      <dl>
+                        <dt><code>rdf:type</code></dt>
+                        <dd>A class whose URI formed by concatenating <code>http://www.w3.org/ns/iana/media-types/</code>, the IANA Media Type [<cite><a class="bibref" href="#bib-iana-media-types">IANA-MEDIA-TYPES</a></cite>] of the resource, and <code>#Resource</code>.</dd>
+                        <dt><code>stat:size</code></dt>
+                        <dd>A non-negative integer giving the size of the resource in bytes.</dd>
+                        <dt><code>dcterms:modified</code></dt>
+                        <dd>The time when the resource was last modified.</dd>
+                        <dt><code>stat:mtime</code></dt>
+                        <dd>The Unix time when the resource was last modified.</dd>
+                      </dl>
+
+                      <div class="note" id="authoritative-resource-data-logically-part-of-container-description" inlist="" rel="schema:hasPart" resource="#authoritative-resource-data-logically-part-of-container-description">
+                        <h5 property="schema:name"><span>Note</span>: Authoritative Resource Data Logically Part of Container Description</h5>
+                        <div datatype="rdf:HTML" property="schema:description">
+                          <p>Authoritative data about contained resources is logically part of the container. Servers are encouraged to consider omitting authoritative data about a contained resource when agent is unauthorized to read the contained resource.</p>
```

Comment:
MAY? SHOULD?

Reply:
If the representation contents may change based on who is viewing the resource, what are the implications for caching?

Reply:
If the following is not agreeable, the advisement need not be in the Note:

I wanted to have an initial acknowledgement of the ongoing considerations until opposing stances are reconciled or a compromise is reached. It is an advisement for servers indicating the case in which they can consider omitting information - this is a legitimate concern for "mission critical systems". The advisement entails that omitting information based on authorization would require the server to do more work and handle the implications - however, this is currently not described in the specification.

Reply:
Yeah, it is probably better left out and dealt with a different mechanism.

Reply:
Clarified the Note in https://github.com/solid/specification/pull/352/commits/f599dd5d9df818ecaa1217359c45ba66a88d5e04 and https://github.com/solid/specification/pull/352/commits/98d2db4ec1d0438cd455163748df413f967cdb36